RetailCoder.VBE/UI/RubberduckUI.fr.resx

```diff
@@ -898,7 +722,7 @@ Attention: les valeurs personnalisées seront perdues.    Le fichier original se
     <value>&amp;Déplacer la déclaration</value>
   </data>
   <data name="IndentCurrentProject" xml:space="preserve">
-    <value>Current Project</value>
+    <value>Projet courant</value>
```

Comment:
Not on you, but this should probably be `IndentActiveProject`, to respect the MS-terminology.